yarn.lock

```diff
@@ -123,6 +123,10 @@
     exec-sh "^0.3.2"
     minimist "^1.2.0"
 
+"@eclipse-che/api@latest":
+  version "7.5.0-SNAPSHOT"
```

Comment:
looks strange

Reply:
Hmm. It is really latest version... https://www.npmjs.com/package/@eclipse-che/api

Reply:
Nothing strange. I just used `latest` in package json, so it was resolved to `7.5.0-SNAPSHOT`

Reply:
Why do we need this at all?

Reply:
I think @mmorhun wants to use dts types for types

Reply:
Yes, I think it is good to have response structure typings generated from Che servers DTOs

Reply:
But is it outdated version?

Reply:
@azatsarynnyy @vparfonov  Do we have newer version of the @eclipse-che/api or release mechanizm become broken after moving to the centos ci?

Reply:
@AndrienkoAleksandr do you know where's the repo with the sources of `@eclipse-che/api`?

Reply:
I think publish here https://github.com/eclipse/che/tree/master/typescript-dto

Reply:
Sorry @AndrienkoAleksandr but I have no idea who is responsible now for `@eclipse-che/api`. Try to ask in Eclipse Che MM channel tomorrow.

Reply:
I think it should a part of release process.

Reply:
Let's not use outdated version.

Reply:
I don't think there were some major changes. But we need to raise this question.

Reply:
I've create issue for this problem: https://github.com/eclipse/che/issues/16436